gulpfile.babel.js

```diff
@@ -111,8 +113,15 @@ function reload(done) {
  */
 export function php() {
 	config = requireUncached('./dev/config/themeConfig.js');
+	// Check if theme slug has been updated.
+	let newThemeSlug = false;
+	if ( config.theme.slug !== themeSlug ) {
+		newThemeSlug = true;
+		themeSlug = config.theme.slug;
+	}
 	return gulp.src(paths.php.src)
-	.pipe(newer(paths.php.dest))
+	// If theme slug has not been updated, run task on changed files only.
+	.pipe(gulpif(!newThemeSlug, newer(paths.php.dest)))
```

Comment:
We don't need this line of code as it is preventing a rebuild in the following scenario:

1. Files already exist, i.e. where previously built.
2. AND theme's slug and/or name changed in the themeConfig.js file.
3. AND when starting gulp again via `npm run build`.


Removing it fixes the above scenario.  All scenarios do cause the PHP files to rebuild. WooHoo!

Reply:
The idea here is for the PHP task to only run on files it needs to run on. The newer method ensures no unnecessary builds. Removing it results in all php files being rebuilt every time a single php file is altered which is resource intensive and unnecessary.

Reply:
To take into account:

1. On script load, i.e. first run.
2. On name or slug change

we can modify this line of code to be:

```
	// If not a rebuild, then run tasks on changed files only.
	.pipe(gulpif(!isRebuild, newer(paths.php.dest)))
```